testcontainers/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "testcontainers"
-version = "0.5.0"
+version = "0.5.1"
```

I think we also want to make a minor version bump on this one. Even though you are allowed to do almost anything prior to 1.0, this is still quite rough to deliver breaking changes with a patch version :D 

```suggestion
version = "0.6.0"
```